states/game_state.py

```diff
@@ -0,0 +1,19 @@
+import pygame
+from observer import Observer
+
+
+class GameState:
```

Comment:
I feel like there should some interface IGameState, that specifies abstract methods like `handle_events`, `update` and so on. Then there should be some `BaseGameState`, which implements `attach` and is then inherited by (for example) `MenuState`